pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -1338,12 +1276,17 @@ metadata:
 spec:
   meta: |
     {{- $jivapodsns := .TaskResult.jivapodsinopenebsns.ns | default .Volume.runNamespace -}}
+    {{- $replicaDeployList := .TaskResult.deletelistrep.names | split " " -}}
```

Comment:
should we do split or splitList? what is the difference?

Reply:
`split` and `splitList` are function from the sprig library. `split` is used to get a numbered map and `splitList` is used to get a plain list of elements. In this case using anyone of them is fine.